hello-vulkan/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+#[[
+ Copyright (C) 2022 The Android Open Source Project
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+#]]
+
+cmake_minimum_required(VERSION 3.18.1)
+project(hellovkjni)
+
+# Include the GameActivity static lib to the project.
+find_package(game-activity REQUIRED CONFIG)
+set(CMAKE_SHARED_LINKER_FLAGS
+    "${CMAKE_SHARED_LINKER_FLAGS} -u \
+    Java_com_google_androidgamesdk_GameActivity_initializeNativeCode")
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# Now build app's shared lib
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -Wall")
```

Comment:
nit: might add -Werror too; if there is any errors, fix them; then remove -Werror?

Reply:
👍 Good shout!